stubs/Symfony/Component/Form/AbstractType.stub

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Symfony\Component\Form;
+
+/**
+ * @template TData
```

Comment:
Every time a new class is made generic, the class name should be added here: https://github.com/phpstan/phpstan-symfony/blob/1da7bf450c6b351fec08ca0aa97298473d4f6ab3/extension.neon#L14-L18 (to make sure users are not bothered by "Function foo() has parameter $hw with generic class HelloWorld but does not specify its types: T" errors.

This list will be made empty with the next major version.

Reply:
Added in [da2cb98](https://github.com/phpstan/phpstan-symfony/pull/337/commits/da2cb981eb40944ba69a93595b7f4e3dedd2b40b)